docs/iris/src/developers_guide/code_format.rst

```diff
@@ -14,7 +14,10 @@ and then manually run from the root of the Iris repository::
   $ black .
   $ flake8 .
 
-Alternatively, we recommend using `pre-commit <https://pre-commit.com/>`_ to automatically run ``black`` and ``flake8``
+Alternatively, ``black`` and ``flake8`` may be installed using ``conda`` instead. Note that, ``black`` also offers
+`editor integration <https://black.readthedocs.io/en/stable/editor_integration.html#editor-integration>`_.
+
+As a developer workflow, we instead recommend using `pre-commit <https://pre-commit.com/>`_ to automatically run ``black`` and ``flake8``
```

Comment:
"As a developer workflow" reads a bit clunkily. It almosts sounds as if you are a developer workflow offering a recomendation...